.circleci/config.yml

```diff
@@ -0,0 +1,305 @@
+
+  store_package_artifacts: &store_package_artifacts
+    store_artifacts:
+      path: *packages_workspace
+      destination: /
+
+  deploy_steps: &deploy_steps
+    - checkout
+    - attach_workspace:
+        at: *packages_workspace
+    - run:
+        name: Install PIP Requirements
+        command: |
+          pip install --user -r deployment/ansible/pip-requirements.txt
+    - run:
+        name: Deploy
+        environment:
+          PACKAGES_DIR: *packages_workspace
+        command: |
+          export PACKAGE=${PACKAGES_DIR:?}/epoch-$(cat VERSION)-linux.tar.gz
+          cd deployment/ansible
+          ansible-playbook -i inventory/openstack.yml --limit="epoch:&${DEPLOY_ENV:?}" \
+            --extra-vars "package=${PACKAGE:?} env=${DEPLOY_ENV:?} \
+              tester_keys_password='${tester_keys_password:?}' datadog_api_key=${datadog_api_key:?} datadog_app_key=${datadog_app_key:?}" \
+            deploy.yml
+
+jobs:
+  build:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - run:
+          name: Build
+          command: ./rebar3 as test do release
+      - save_cache:
+          key: build-cache-v1
+          paths:
+            - "_build"
+            - "apps/aecuckoo/c_src"
+            - "apps/aecuckoo/priv/bin"
+            - "apps/aecuckoo/priv/lib"
+      - save_cache:
+          key: *rebar_cache_key
+          paths:
+            - .cache/rebar3
+
+  test:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - run:
+          name: Test
+          command: |
+            epmd -daemon
+            make test
+      - store_artifacts:
+          path: _build/test/logs
+      - store_artifacts:
+          path: _build/test/logs/latest.sync/*/log/*.log
+
+  eunit:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - run:
+          name: Test
+          command: |
+            epmd -daemon
+            make eunit
+
+  dialyzer:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - restore_cache:
+          keys:
+            - dialyzer-cache-v2-{{ .Branch }}-{{ .Revision }}
+            - dialyzer-cache-v2-{{ .Branch }}-
+            - dialyzer-cache-v2-
+      - run:
+          name: Update dialyzer PLT
+          command: make dialyzer-install
+      - save_cache:
+          key: dialyzer-cache-v2-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - _build/default/rebar3_20.2.2_plt
+      - run: make dialyzer
+
+  linux_package:
+    <<: *container_config
+    steps:
+      - checkout
+      - run: id
+      - *build_package
+      - *test_package
+      - *store_package_artifacts
+      - persist_to_workspace:
+          root: *packages_workspace
+          paths:
+            - "*.tar.gz"
+
+  osx_package:
+    macos:
+      xcode: "9.0"
```

Comment:
We need to know package name before the release if we keep package name inside release notes.

Reply:
According to the CCI docs it should be 10.12.6, see
https://circleci.com/docs/1.0/build-image-macos/#selecting-xcode-version-and-operating-system and https://circle-macos-docs.s3.amazonaws.com/image-manifest/build-282/index.html

Reply:
v2.0 doc: https://circleci.com/docs/2.0/testing-ios/#supported-xcode-versions